test/index.test.js

```diff
@@ -111,6 +111,31 @@ describe('Weather Module', () => {
       expect(weatherStatus).to.deep.equal(expected);
     });
 
+    it('should round to correct precision', async () => {
+      axiosStub.withArgs(sinon.match(new RegExp(`${config.baseUrl}/\\d+/starfwhm.*`)))
+        .returns(mockRespond(156.498));
+      axiosStub.withArgs(sinon.match(new RegExp(`${config.baseUrl}/\\d+/temperature.*`)))
+        .returns(mockRespond(-1.4652419));
+      axiosStub.withArgs(sinon.match(new RegExp(`${config.baseUrl}/\\d+/skytemperature.*`)))
+        .returns(mockRespond(-15.647585));
+      axiosStub.withArgs(sinon.match(new RegExp(`${config.baseUrl}/\\d+/rainrate.*`)))
+        .returns(mockRespond(1.46524));
+      axiosStub.withArgs(sinon.match(new RegExp(`${config.baseUrl}/\\d+/pressure.*`)))
+        .returns(mockRespond(5463151.3));
```

Comment:
Should the precision effect whole numbers as well as decimals? As in, do you really want `5463151.3` to get changed to `5463200`?

Reply:
Yeah, I think it should.  Whether or not the number has that many digits on the left side could be a matter of what units are being used (meters vs centimeters).  Under that context, the position of the decimal point is fairly arbitrary.  Because of that, I do think even whole numbers should be rounded if you're okay with that.

Reply:
Sure, just making sure that was your intention.